cves/CVE-2012-0858.yml

```diff
@@ -193,8 +196,10 @@ autodiscoverable:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  answer_note:
-  answer:
+  answer_note: use-after-free would find this because the issue was an invalid free.
+    Use-after-free is just when memory is used after it is freed. 
```

Comment:
Maybe a little bit more explanation as to how you could go about using 'use-after-free' to find this issue could be helpful here.

Reply:
Better description on why use-after-free would be helpful to finding this bug